lib/index.js

```diff
@@ -8,11 +8,17 @@ const path = require('path');
 const sass = require('node-sass');
 const argv = require('argv-parse');
 const verify = require('./verify');
+const {winston} = require('./logging');
```

Comment:
If you have a logger abstract module, I think we should export `logger` rather than the implementation-specific `winston`

Reply:
Yeah, good point. I will call it `logger`.